lte/gateway/c/core/oai/tasks/mme_app/mme_app_state.cpp

```diff
@@ -54,8 +54,8 @@ void put_mme_nas_state() {
  */
 void clear_mme_nas_state() { MmeNasStateManager::getInstance().free_state(); }
 
-hash_table_ts_t* get_mme_ue_state() {
-  return MmeNasStateManager::getInstance().get_ue_state_ht();
+proto_map_uint32_ue_context_t* get_mme_ue_state() {
```

Comment:
Can you insert a blank line between line 18 and 19 to hightlight the first header as the associated header?